.github/workflows/python-package.yml

```diff
@@ -16,7 +16,25 @@ on:
     branches: [ master ]
 
 jobs:
-  build:
+  cpp-build:
```

Comment:
Extremely minor: I would keep the naming consistent between jobs. So either use dashes underscores.